cmd/shell.go

```diff
@@ -316,3 +317,41 @@ func LoadCert(path string) (cert []byte, err error) {
 	cert = block.Bytes
 	return
 }
+
+var openConnections int64
+
+// HandlerTimer monitors HTTP performance and sends the details to StatsD.
+func HandlerTimer(handler http.Handler, stats statsd.Statter, prefix string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Please push this over a new library called stats or metrics or handlers (probably the first) and write a test for it if there isn't one already.